srcts/src/components/_documentObserver.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
I'm starting to wonder if we should just remove this (or comment it out completely) since, it seems, if we need to avoid `<script>` tags for initialization, we should use a web component (instead of `MutationObserver` on the document?)

Reply:
I went ahead and removed it; we can always pull it out of the commit history. 0d6922421eae97556e91dfd06943c91554daf290